tests/nightly/test_large_array.py

```diff
@@ -1791,6 +1792,29 @@ def test_sparse_dot():
     assert out.shape == (2, 2)
 
 
+def test_linalg_operators():
+    def check_syrk_batch():
+        # test both forward and backward
+        # batch syrk will be applied to the last two dimensions
+        A = nd.zeros((2, LARGE_SQ_X, LARGE_SQ_X))
+        for i in range(LARGE_SQ_X):
+            A[0,i,i] = 1
+            A[1,i,i] = 0.1
+        A.attach_grad()
+        with mx.autograd.record():
+            out = nd.linalg.syrk(A, alpha=2, transpose=False)
+        for i in range(LARGE_SQ_X):
```

Comment:
You can check in 2 places in (0,y,y) and (1,y,y). No need to check in 70000 locations